### ISSUE TITLE
How the app compared to others

### ISSUE THREAD

**Issue by dimqua** [Teester/Whats-Nearby#48]
Could you please explain how this app compared to similar apps like StreetComplete? I believe the latter one does exactly the same. Am I right?

**Comment by Teester** [comment]
Not quite.  While both apps ask questions about osm elements, What's Nearby prompts you to answer questions automatically, while StreetComplete requires you to open the app.

If you are familiar with the notifications that Google Maps sometimes gives asking you to answer questions about a place you have recently been, then What's Nearby is similar, though with questions relevant to osm, obviously.

See #6 for more discussion.
